src/clim2parquet/tools.py

```diff
@@ -206,8 +401,13 @@ def _files_to_parquet(files: list[Path], to: str) -> None:
     # NOTE: function will error if there are no files
     for file in files:
         df = pd.read_csv(file, sep=",", header=0)
+        _add_admin_unit_id(
+            df,
+            _get_admin_data(str(file), admin_level, gadm_version),
+            admin_unit_ids,
+        )
         data_list.append(df)
 
     df = pd.concat(data_list)
-    table = pa.Table.from_pandas(df)
+    table = pa.Table.from_pandas(df, preserve_index=False)
```

Comment:
I had a look at some compression effects here, and for the file generated for `test_output_format_lvl_1` the results are (in bytes)

64172 - BROTLI-9
64222 - BROTLI
65768 - zstd-9
68807 - zstd
71998 - GZIP
71998 - GZIP-9
89339 - LZ4-9
94274 - current (snappy)
96688 - LZ4
242094 - csv

so we're currently at ~2.5x compression but we can get to 3.8; this feels worth implementing but it is probably worth doing in the next PR probably against a run against a larger set of real outputs?

Docs: https://arrow.apache.org/docs/python/generated/pyarrow.parquet.write_table.html

Reply:
Sure - I'll pop that in as an issue.